mne/decoding/transformer.py

```diff
@@ -172,6 +172,34 @@ def transform(self, epochs_data, y=None):
         return _sklearn_reshape_apply(self._scaler.transform, True,
                                       epochs_data, y)
 
+    def fit_transform(self, epochs_data, y=None):
+        """Fit to data, then transform it.
+
+        Fits transformer to epochs_data and y and returns a transformed version 
+        of epochs_data.
+
+        Parameters
+        ----------
+        epochs_data : array, shape (n_epochs, n_channels, n_times)
+            The data.
+        y : None | array, shape (n_epochs,)
+            The label for each epoch.
+            Defaults to None.
+
+        Returns
+        -------
+        X : array, shape (n_epochs, n_channels, n_times)
+            The data concatenated over channels.
+
+        Notes
+        -----
+        This function makes a copy of the data before the operations and the
+        memory usage may be large with big data.
+        """
+        return _sklearn_reshape_apply(self._scaler.fit_transform, True,
+                                      epochs_data, y)
```

Comment:
> why not just returning
> self.fit(X, y).transform(X)

Yep, we can return this, I thought that functions `transform`, `fit`, `inverse_transform` should look like the same. But the answer to weather it is required? is definitely not because reshaping will be taken care in `fit` and `transform` functions. 
If you think `self.fit(X, y).transform(X)` to be more correct than I can change it.

> is this method covered by a test?

I think yes. See [this](https://github.com/mne-tools/mne-python/blob/master/mne/decoding/tests/test_transformer.py#L78)